packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1378,13 +1378,20 @@ async function handler(
 
       // loop through all non-fixed hosts and get the lucky users
       while (luckyUserPool.length > 0 && luckyUsers.length < 1 /* TODO: Add variable */) {
-        const newLuckyUser = await getLuckyUser("MAXIMIZE_AVAILABILITY", {
-          // find a lucky user that is not already in the luckyUsers array
-          availableUsers: luckyUserPool.filter(
-            (user) => !luckyUsers.concat(notAvailableLuckyUsers).find((existing) => existing.id === user.id)
-          ),
-          eventTypeId: eventType.id,
-        });
+        const freeUsers = luckyUserPool.filter(
+          (user) => !luckyUsers.concat(notAvailableLuckyUsers).find((existing) => existing.id === user.id)
+        );
+        const userId = originalRescheduledBooking && originalRescheduledBooking.userId;
```

Comment:
```suggestion
        const originalBookingUserId = originalRescheduledBooking && originalRescheduledBooking.userId;
```

NIT: better variable naming for clarity